canvas/base.go

```diff
@@ -11,14 +11,15 @@ import (
 	"sync"
 
 	"fyne.io/fyne/v2"
+	"fyne.io/fyne/v2/internal/async"
```

Comment:
Is `atomic` not a better name for this internal package? Yes it conflicts with the Go builtin `atomic` package, but in most places (including here) we don't need to import the stdlib atomic package, and the naming is much more clear that IMO it is worth it if we occasionally have to alias the import as `fyneAtomic` or similar

Reply:
I figured that we already have the package there so it is better to keep it there and also avoid the conflict?

Reply:
OK yeah, somehow in my mind I was thinking that internal/async was a new package with this PR